pkg/content_move.go

```diff
@@ -0,0 +1,30 @@
+package pkg
+
+import (
+	"github.com/wttech/aemc/pkg/common/filex"
+	"github.com/wttech/aemc/pkg/content"
+)
+
+type Mover struct {
+	config *content.Opts
+}
+
+func NewMover(config *content.Opts) *Mover {
+	return &Mover{
+		config: config,
+	}
+}
+
+func (c Mover) Move(scrPackageManager *PackageManager, descPackageManager *PackageManager, filter string, clean bool) error {
+	err := NewDownloader(c.config).Download(scrPackageManager, "/tmp/aemc_content", filter, clean)
```

Comment:
don't use hardcoded `/tmp` ; it will not work on Windows

in AEMC there is dedicated tmp dir defined in base opts